engine/src/mbliphonescroller.mm

```diff
@@ -752,7 +752,12 @@ bool MCScrollViewGetContentOffset(UIScrollView *p_view, int32_t &r_x, int32_t &r
 	t_view = [[UIScrollView alloc] initWithFrame: CGRectMake(0, 0, 0, 0)];
 	if (t_view == nil)
 		return nil;
-	
+
+    if (@available(iOS 11.0, *)) {
```

Comment:
@livecodeali I think you also need to wrap this in a pre-processor macro because `UIScrollViewContentInsetAdjustmentNever` won't be available for some of the SDKs we still build against:
```
#if __IPHONE_OS_VERSION_MAX_ALLOWED > 110000
```